projet/users/serializers.py

```diff
@@ -48,3 +49,27 @@ class Meta:
             "image",
             "bio",
         )
+
+
+class ContributorSerializer(serializers.ModelSerializer):
+    profile_image = serializers.ImageField(
+        source="user.profile.image", read_only=True, required=False
+    )
```

Comment:
Again it is not required so required had to be set to false otherwise will cause some issues.